internal/core/core.go

```diff
@@ -13,6 +13,7 @@ type TransitData struct {
 	Value any
 }
 
+// NewTransitChannel ... Builds new tranit channel
```

Comment:
Was this something we missed before? If so, is it possible to make it linter-enforced? Can add a new issue if so

Reply:
Let's ad an issue for this one; I looked into linting policies for go 1.19 but couldn't find any specific to function comment enforcement. Weirdly enough it was a native go-lint scan for past golang versions.

Reply:
Function made, feel free to edit: #40